Makefile.in

```diff
@@ -1,7 +1,7 @@
-# Makefile.in generated by automake 1.15.1 from Makefile.am.
+# Makefile.in generated by automake 1.16.1 from Makefile.am.
```

Comment:
I guess we should be stick to 1.15.1?

Reply:
dejan needs to be using 1.15.1, which is our standard.